### PR TITLE
autorestic 1.5.5

### PR DESCRIPTION
**Pull request by Porkepix**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
It looks like this refuses to generate completions when the config file is missing. That seems like a bug, since there's no reason it should need one to generate completions.

**Comment by cupcakearmy** [comment]
Yes, my bad. Should be fixed in 1.5.3 https://github.com/cupcakearmy/autorestic/releases/tag/v1.5.3

File: Formula/autorestic.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Autorestic < Formula
   desc "High level CLI utility for restic"
   homepage "https://autorestic.vercel.app/"
-  url "https://github.com/cupcakearmy/autorestic/archive/v1.5.1.tar.gz"
-  sha256 "f0ad322b51408d5a47435923b9cc4858c4ba030bc0e7ad192ec40c5e7cc80e67"
+  url "https://github.com/cupcakearmy/autorestic/archive/v1.5.4.tar.gz"
```
Comment (carlocab) [inline Formula/autorestic.rb:4]:
```suggestion
  url "https://github.com/cupcakearmy/autorestic/archive/v1.5.5.tar.gz"
```

Comment (carlocab) [reply Formula/autorestic.rb]: The hash probably also needs updating @chenrui333

**Review by cho-m — APPROVED** [review]

**Pull request closed without merging**